sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleDto.java

```diff
@@ -106,6 +106,8 @@
 	private String requestedOtherPathogenTests;
 	private String requestedOtherAdditionalTests;
 
+	private PathogenTestDto defaultTest;
```

Comment:
Please remove this field; it doesn't serve any purpose after the sample has been initially created. We should save the pathogen test independently instead during sample creation instead.